src/main/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/PractitionerMapper.java

```diff
@@ -1,32 +1,50 @@
 package uk.nhs.digital.nhsconnect.lab.results.translator.mapper;
 
+import org.apache.commons.lang3.StringUtils;
 import org.hl7.fhir.dstu3.model.Practitioner;
 import org.springframework.stereotype.Component;
+
+import lombok.RequiredArgsConstructor;
 import uk.nhs.digital.nhsconnect.lab.results.model.edifact.Message;
-import uk.nhs.digital.nhsconnect.lab.results.model.edifact.RequesterNameAndAddress;
 import uk.nhs.digital.nhsconnect.lab.results.model.edifact.segmentgroup.InvolvedParty;
+import uk.nhs.digital.nhsconnect.lab.results.utils.UUIDGenerator;
 
 import java.util.Optional;
 
 @Component
+@RequiredArgsConstructor
```

Comment:
Surprised this doesn't seem to need autowiring?
```suggestion
@RequiredArgsConstructor(onConstructor_ = @Autowired)
```

Reply:
Added in 4bbb811